exercises/hello-world/lib/hello_world.dart

```diff
@@ -1,5 +1,5 @@
 class HelloWorld {
-  String hello([String name]) {
+  String hello() {
```

Comment:
Not within the scope of this work. If this is a bug or oversight from previous work, then I think the right step would be to create an issue or PR specificaly to resolve.